src/sqlfluff/dialects/dialect_spark3.py

```diff
@@ -1515,6 +1515,29 @@ class SamplingExpressionSegment(BaseSegment):
     )
 
 
+@spark3_dialect.segment()
+class LateralViewClauseSegment(BaseSegment):
+    """A `LATERAL VIEW` like in a `FROM` clause.
+
+    https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-lateral-view.html
+    """
+
+    type = "lateral_view_clause"
+
+    match_grammar = Sequence(
+        Indent,
+        "LATERAL",
+        "VIEW",
+        Ref.keyword("OUTER", optional=True),
+        Ref("FunctionSegment"),
+        AnyNumberOf(
+            Ref("AliasExpressionSegment", optional=True),
+            min_times=1,
+        ),
```

Comment:
Are these not contradictory? If it's optional isn't `min_times` actually 0?

Could this not be simplified to:

```suggestion
        AnyNumberOf(Ref("AliasExpressionSegment")),
```